pagopa/pn-bff
# feat(pn-11377): github actions for dependencies management

## Short description

Created action to automatically update the micro-service dependencies

## List of changes proposed in this pull request

- Added new action under .github/workflows/actions
- Added new workflow under .github/workflows

## How to test

Merge the pr -> go to actions tab -> run the action and check if a pr with updated dependencies is created
